app/code/core/Mage/Adminhtml/Block/System/Config/Form.php

```diff
@@ -637,9 +641,9 @@ public function getScope()
      */
     public function getScopeLabel($element)
     {
-        if ($element->show_in_store == 1) {
+        if ((string)$element->show_in_store == '1') {
```

Comment:
How about
```suggestion
        if ((int)$element->show_in_store === 1) {
```
since we expect `int 1|0`.